Aptoide/aptoide-client-v8
# [Feature/ASV-1332] http to https migration (pnp and buzz)

**What does this PR do?**

   Enables usage of https in production for the pnp and buzz hosts.
   pnp and buzz are now also toolbox compliant. For the buzz case, because the https port (10002) doesn't support http requests, the toolbox+aptoide scheme logic needed to be changed. Now not only http/https changes as the search host port also changes (10001 vs 10002) being the the latest the ssl compliant one.
   

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] SearchHostProvider.java

**How should this be manually tested?**

  pnp is used in the notifications pull (campaigns), so first running the app will allow you to check in charles if it is https or not.
  buzz is used for the seach suggestions, so again use charles to check protocol. 

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1332](https://aptoide.atlassian.net/browse/ASV-1332)

**Code Review Checklist**
